src/axom/slic/interface/slic_shroud.yaml

```diff
@@ -33,6 +32,8 @@ format:
 declarations:
 - decl: namespace message
   # MessageLevel.h
+  options:
+    flatten_namespace: True
```

Comment:
@gzagaris The `flatten_namespace` option here prevents the message namespace from becoming a module on its own.  It preserves more of the old behavior but ends up renaming the enums.  It might be worth exploring removing this option, it's the only place it's used in Axom.

Reply:
@ltaylor16 -- if that's what you recommend as best practice, I think we should remove it.

Reply:
I tested removing the `flatten_namespace` option and it creates a new module `slic_message_mod` with parameters for the enums named `error`, `warning`, etc.  I think this is worse since the names `error` and `warning` are very generic. The C api did not change. Shroud only put them in a different file (one file per namespace).
This code was written before we assumed C++11.  Today it would be better to use an `enum class` to scope the enums instead of a namespace. But that's another issue.
I will add the changed names in the Fortran API to  `RELEASE-NOTES.md`.

Reply:
So are we removing this or not? This would require the Fortran application to have `use slic_message_mod`, which I think would be annoying. I think that what we would want is for the application to be able to simply do `use slic_mod` and get everything, right?

Reply:
Keep the `flatten_namespace`.  The main issue is it gives better enums: `message_error` instead of just `error`.  It's possible to put `use slic_message_mod` inside module slic_mod so users could still just `use slic_mod` to get everything so that is less of an issue.

Reply:
Got it -- Thanks.